processor-api/src/main/java/com/synopsys/integration/alert/processor/api/filter/model/FilterableNotificationWrapper.java

```diff
@@ -29,45 +29,45 @@
 import com.synopsys.integration.blackduck.api.manual.component.VulnerabilityNotificationContent;
 
 public class FilterableNotificationWrapper<T extends NotificationContentComponent> extends ProcessableNotificationWrapper<T> {
-    private final String projectName;
+    private final List<String> projectNames;
     private final List<String> policyNames;
     private final List<String> vulnerabilitySeverities;
 
     public static FilterableNotificationWrapper<VulnerabilityNotificationContent> vulnerability(
         AlertNotificationModel notificationModel,
         VulnerabilityNotificationContent notificationContent,
-        String projectName,
+        List<String> projectNames,
         List<String> vulnerabilitySeverities
     ) {
-        return new FilterableNotificationWrapper<>(notificationModel, notificationContent, projectName, List.of(), vulnerabilitySeverities);
+        return new FilterableNotificationWrapper<>(notificationModel, notificationContent, projectNames, List.of(), vulnerabilitySeverities);
     }
 
     public static FilterableNotificationWrapper<? extends NotificationContentComponent> policy(
         AlertNotificationModel notificationModel,
         NotificationContentComponent notificationContent,
-        String projectName,
+        List<String> projectNames,
         List<String> policyNames
     ) {
-        return new FilterableNotificationWrapper<>(notificationModel, notificationContent, projectName, policyNames, List.of());
+        return new FilterableNotificationWrapper<>(notificationModel, notificationContent, projectNames, policyNames, List.of());
     }
 
     public static FilterableNotificationWrapper<? extends NotificationContentComponent> project(AlertNotificationModel notificationModel, NotificationContentComponent notificationContent, String projectName) {
-        return new FilterableNotificationWrapper<>(notificationModel, notificationContent, projectName, List.of(), List.of());
+        return new FilterableNotificationWrapper<>(notificationModel, notificationContent, List.of(projectName), List.of(), List.of());
     }
 
     public static FilterableNotificationWrapper<? extends NotificationContentComponent> projectless(AlertNotificationModel notificationModel, NotificationContentComponent notificationContent) {
         return new FilterableNotificationWrapper<>(notificationModel, notificationContent, null, List.of(), List.of());
     }
 
-    public FilterableNotificationWrapper(AlertNotificationModel alertNotificationModel, T notificationContent, String projectName, List<String> policyNames, List<String> vulnerabilitySeverities) {
+    public FilterableNotificationWrapper(AlertNotificationModel alertNotificationModel, T notificationContent, List<String> projectNames, List<String> policyNames, List<String> vulnerabilitySeverities) {
```

Comment:
If the static methods are the only methods that are used to create this object then the constructor could be private.

Reply:
I will make this change in my next PR as there may be some further modifications to how these work